src/aws-type-downloader/main.go

```diff
@@ -18,14 +18,8 @@ import (
 )
 
 // These have been skipped due to missing schema: https://github.com/aws/aws-sdk-go-v2/issues/2913
-// These will be reverted when the issue is resolved.
-var skippedResources = map[string]struct{}{
-	"AWS::EC2::LaunchTemplate":   {},
-	"AWS::QuickSight::Analysis":  {},
-	"AWS::QuickSight::Dashboard": {},
-	"AWS::QuickSight::Template":  {},
-	"AWS::SageMaker::Cluster":    {},
-}
+// These will be reverted when the issue is resolved, but the map will be kept for future use if needed.
```

Comment:
Should the comment be updated to reflect what the purpose of this map is/when to use it instead of keeping it linked to this old issue?

Reply:
Updated to: 
`This map is used as a workaround when there are issues with missing schema from AWS CloudFormation specs. Since we do not own AWS schemas, we may have to wait for the proper updates to be made. Any resources added to this map will be skipped during downloads so a previous, working version of that resource schema can be used.`